pulp_container/app/migrations/0004_upload.py

```diff
@@ -14,26 +14,4 @@ class Migration(migrations.Migration):
         ('container', '0003_oci_mediatype'),
     ]
 
-    operations = [
-        migrations.CreateModel(
-            name='Upload',
-            fields=[
-                ('pulp_id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
-                ('pulp_created', models.DateTimeField(auto_now_add=True)),
-                ('pulp_last_updated', models.DateTimeField(auto_now=True, null=True)),
-                ('offset', models.BigIntegerField(default=0)),
-                ('file', models.FileField(max_length=255, null=True, storage=django.core.files.storage.FileSystemStorage(location='/var/lib/pulp/upload/'), upload_to=pulp_container.app.models.generate_filename)),
-                ('size', models.IntegerField(null=True)),
-                ('md5', models.CharField(max_length=32, null=True)),
-                ('sha1', models.CharField(max_length=40, null=True)),
-                ('sha224', models.CharField(max_length=56, null=True)),
-                ('sha256', models.CharField(max_length=64, null=True)),
-                ('sha384', models.CharField(max_length=96, null=True)),
-                ('sha512', models.CharField(max_length=128, null=True)),
-                ('repository', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='uploads', to='core.Repository')),
-            ],
-            options={
-                'abstract': False,
-            },
-        ),
-    ]
+    operations = []
```

Comment:
Please do not modify existing migrations.

Reply:
I was getting errors when I removed the function `generate_filename()`: 
https://github.com/pulp/pulp_container/blob/13800035477c6d43ebb78af33262c934551d5490/pulp_container/app/models.py#L362-L365.
That is why I purged this migration. Do you propose to leave the function there?

Reply:
In that case, i guess yes.